io/io/inc/TThreadMergingServer.h

```diff
@@ -0,0 +1,92 @@
+#ifndef ROOT_TThreadMergingServer
+#define ROOT_TThreadMergingServer
+
+#include "dcqueue.h"
+
+#include "TBits.h"
+#include "TFile.h"
+#include "TFileCacheWrite.h"
+#include "TFileMerger.h"
+#include "TTimeStamp.h"
+
+struct ClientInfo {
+   TFile *fFile; // This object does *not* own the file,
+                 // it will be owned by the owner of the ClientInfo.
+   TString fLocalName;
+   UInt_t fContactsCount;
+   TTimeStamp fLastContact;
+   Double_t fTimeSincePrevContact;
+
+   ClientInfo() : fFile(0), fLocalName(), fContactsCount(0), fTimeSincePrevContact(0) {}
```

Comment:
I meant, it'd be better to initialize the variables in place, i.e. as part of their definitions.